docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss

```diff
@@ -250,7 +255,7 @@
 
 .more-info-list ul li:before,
 .cthp-pdq-label ul li:before,
-.cardBody h4 + ul li:before,
+.cardBody ul li:before,
```

Comment:
Can I poke my nose in and ask why we're making this markup change? These aren't the same selectors on prod, as demonstrated here: 
![image](https://user-images.githubusercontent.com/34315527/56150176-4897c480-5f7c-11e9-9865-051dbd886a2d.png)

Reply:
Oh, I just noticed you set me as a reviewer. So I guess I'm not technically sticking my nose in. :)

Reply:
Speaking with the IA, if for some strange reason a content editor added a list without a list heading and didn't realize how dumb it looks, the list SHOULD be bulleted.

Reply:
Stupid content editors...